lib/screens/conversations_screen.dart

```diff
@@ -0,0 +1,61 @@
+import 'dart:math';
+import 'package:chirp_nets/providers/users.dart';
+import 'package:flutter/material.dart';
+import 'package:provider/provider.dart';
+
+import 'package:chirp_nets/widgets/converstaion_widget.dart';
+import 'package:chirp_nets/models/conversation.dart';
+import 'package:chirp_nets/models/user.dart';
+import 'package:chirp_nets/providers/conversations.dart';
+import 'package:chirp_nets/utils/database.dart';
+
+class ConversationsScreen extends StatelessWidget {
+  void addConversation(Conversations conversationData, User user) {
+    Conversation conv = Conversation(
+      userId: user.id,
+      name: Random().nextInt(100).toString(),
```

Comment:
What does name do? I assumed it was the name of the groupchat or the name of the person being messaged, but it appears to be a number

Reply:
its the name of a conversation. that was here as a placeholder until adding conversations was done